gcc/rust/hir/tree/rust-hir-item.h

```diff
@@ -2801,6 +2802,8 @@ class ImplBlock : public VisItem, public WithInnerAttrs
   // Returns whether inherent impl block has inherent impl items.
   bool has_impl_items () const { return !impl_items.empty (); }
 
+  bool is_unsafe () const { return unsafe; }
```

Comment:
might not be necesary due to `if (unsafe_context.is_in_context ())` ?

Reply:
I think the same comment left https://github.com/Rust-GCC/gccrs/pull/3078#discussion_r1669367590 would apply to this as well